documentation/book/con-custom-resources-example.adoc

```diff
@@ -31,7 +31,9 @@ spec: <2>
     - kt <3>
   additionalPrinterColumns: <4>
       # ...
-  validation: <5>
+subresources: 
```

Comment:
I think this is mis-aligned. You should add 2 more spaces before the `subresources:` and `status: {}`.